README.md

```diff
@@ -154,6 +154,19 @@ ObjectMapper mapper = new ObjectMapper();
 String requiredJson = mapper.writeValueAsString(query);
 ```
 
+``` java
+DruidConfiguration config =  DruidConfiguration
+               .builder()
+               .host("druid.io")
+               .endpoint("druid/v2/")
+               .build();
+
+DruidClient client = new DruidJerseyClient(druidConfiguration);
+client.connect();
+List<DruidResponse> responses = client.query(query, DruidResponse.class);
+client.close();
+```
```

Comment:
One example with how to enable and disable gZip with help. Also one example to get query result as POJO

Reply:
Also mention the expected java version say Java 7 or 8

Reply:
Will add Gzip in wiki.

Reply:
Updated README.md with another section "Prerequisite" under Getting started